config/locales/en.yml

```diff
@@ -225,6 +225,9 @@ en:
       transparency:
         one: Transparency data
         other: Transparency data
+      world_location_news_article:
```

Comment:
`document_type` is always "news_article" in the schema examples. Is this string needed?
If we do need to keep it then we need to generate the translations for the other locales too, as this format can have translations.